app/models/portfolio.rb

```diff
@@ -13,8 +13,9 @@
 class Portfolio < ApplicationRecord
   acts_as_tenant(:tenant)
 
-  validates_presence_of :name
-  validates_presence_of :description
+  validates :name, presence: true, :uniqueness => { :scope => :tenant_id }
```

Comment:
@lindgrenj6 @syncrou What is the API response when the validation fails?

Reply:
It is a 422, name already taken. Adding a test for it.